cves/kernel/CVE-2020-24394.yml

```diff
@@ -177,8 +174,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    The vulnerability is dealing with setting permissions of a file system, 
```

Comment:
I would change this to active voice instead of "is dealing", "deals" might sound better